### PR TITLE
cleaned up product names

### PR DESCRIPTION
**Pull request by abuehrle**
@bboreham @awh did another pass and fixed the product name to say Weave Net instead of just weave. 

Also, do we want to replace the ascii art in IPAM and use a new diagram or did you want me to include the png representation of that diagram. Wasn't sure. 

Also still need to add the renaming of the interface by weave. Where to include that?

### CODE REVIEW

File: site/ipam/overview-init-ipam.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ through three states: 'deferred', 'waiting' and 'achieved':
   themselves successfully
 * 'achieved' - consensus achieved; allocations proceed normally
 
-Finally, you may launch some peers as election observers using the
+Finally, you some peers can be launched as election observers using the
```
Comment (awh) [inline site/ipam/overview-init-ipam.md]: This sentence has become slightly broken

File: site/ipam/stop-remove-peers-ipam.md [hunk 2]
```diff
@@ -5,29 +5,30 @@ layout: default
...
+If, however, you run `weave reset` this removes the peer from the network so
+if Weave Net is run again on that node it will start from scratch.
 
 For failed peers, the `weave rmpeer` command can be used to
-permanently remove the ranges allocated to said peer.  This will allow
-other peers to allocate IPs in the ranges previously owner by the rm'd
-peer, and as such should be used with extreme caution - if the rm'd
+permanently remove the ranges allocated to said peer.  This allows
+other peers to allocate IPs in the ranges previously owned by the removed peer
+peer, and as such should be used with extreme caution - if the removed
```
Comment (awh) [inline site/ipam/stop-remove-peers-ipam.md]: Extraneous 'peer'

File: site/using-weave/service-management.md [hunk 3]
```diff
@@ -48,7 +48,7 @@ Weave hosts, regardless of where the actual application containers are located.
 Using the netcat service example described in 
 [Deploying Applications](/site/using-weave/deploying-applications.md), and expanding upon it, you now decide to add a third containerized netcat service. This additional netcat service runs on `$HOST3`, and listens on port 2211, but it is not on the Weave network. 
 
-An additional caveat is that `$HOST3` can only be reached from `$HOST1`, which is not accessible via `$HOST2`. Nonetheless, you still need to make the `$HOST3` service available to an application that is running in a container on `$HOST2`.
+An additional caveat is that `$HOST3` can only be reached from `$HOST1`, which is not accessible via `$HOST2`. Nonetheless, you still need to make a `$HOST3` service available to an application that is running in a container on `$HOST2`.
```
Comment (awh) [inline site/using-weave/service-management.md]: I think this should be 'the' and not 'a' - the sentence is referring to the particular service instance in the previous paragraph, not an arbitrary service

File: site/weave-docker-api/automatic-discovery-proxy.md [hunk 4]
```diff
@@ -35,8 +35,7 @@ For example, you can launch the proxy using all three flags, as follows:
 (for example, `$1`). For further details on the regular expression syntax see
 [Google's re2 documentation](https://github.com/google/re2/wiki/Syntax).
 
-After launching the Weave proxy with these flags, running a container named `aws-12798186823-foo` without labels results in
-weavedns registering the hostname `my-app-foo` and not `aws-12798186823-foo`.
+After launching the Weave Net proxy with these flags, running a container named `aws-12798186823-foo` without labels results in weavedns registering the hostname `my-app-foo` and not `aws-12798186823-foo`.
```
Comment (awh) [inline site/weave-docker-api/automatic-discovery-proxy.md]: For my edification - is 'weavedns' the new standardised term?

**Comment by awh** [comment]
Completed a review pass @abuehrle

**Comment by abuehrle** [comment]
Yes, I was told we don't want to capitalize items that are not products.

**Pull request merged**